cypress/integration/rendering/sequencediagram.spec.js

```diff
@@ -3,6 +3,45 @@
 import { imgSnapshotTest, renderGraph } from '../../helpers/util';
 
 context('Sequence diagram', () => {
+  it('should render a sequence diagram with boxes', () => {
```

Comment:
Appreciate that you added a rendering test. These are crucial for regression testing.